integrations/gcp/src/commonMain/kotlin/com/xebia/functional/xef/gcp/GcpScopeExtensions.kt

```diff
@@ -0,0 +1,19 @@
+package com.xebia.functional.xef.gcp
+
+import com.xebia.functional.xef.conversation.AiDsl
+import com.xebia.functional.xef.conversation.Conversation
+import com.xebia.functional.xef.llm.Chat
+import com.xebia.functional.xef.prompt.Prompt
+import kotlinx.coroutines.flow.Flow
+
+@AiDsl
+suspend fun Conversation.promptMessageGcp(
```

Comment:
These functions should not have the `Gcp` suffix.
A user may choose to add the name by qualified import if it wants to call it that way, but in general, this should be just `promptMessage`, the same for the streaming function.